servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -38,11 +38,12 @@
 import tachyon.Version;
 import tachyon.conf.TachyonConf;
 import tachyon.metrics.MetricsSystem;
+import tachyon.security.authentication.AuthenticationFactory;
 import tachyon.thrift.MasterService;
 import tachyon.underfs.UnderFileSystem;
 import tachyon.util.CommonUtils;
-import tachyon.util.network.NetworkAddressUtils;
 import tachyon.util.ThreadFactoryUtils;
+import tachyon.util.network.NetworkAddressUtils;
```

Comment:
Can you check why this line shows up in the diff? They look identical and might embed some special character in either of them.